WeatherApp/Model/WeatherModel.swift

```diff
@@ -8,43 +8,71 @@
 
 import Foundation
 
-struct WeatherModel: Codable {
+struct WeatherModel {
     let cityName: String
     let dayForecast: [DayForecast]
-    var fromLocation: Bool
-    
-    struct DayForecast: Codable {
```

Comment:
Ja zazwyczaj nie zagnieżdzam struktur, bo może być używana w innym kontekście. Ale to jest do dogadania w zespole. Trzeba ustalić konwencję i się jej trzymać :)